src/main/java/jenkins/plugins/logstash/persistence/LogstashIndexerDao.java

```diff
@@ -42,7 +42,8 @@
     REDIS,
     RABBIT_MQ,
     ELASTICSEARCH,
-    SYSLOG
+    SYSLOG,
+    LOGZIO
```

Comment:
this never existed in old version, so don't add it

Reply:
I need it If I want to add a new endpoint, no?

Reply:
You don't need it. Guess why it is deprecated.
Your Configuration class is the extension point. The old way was not using extensions so when you needed another endpoint you would have to change the plugin. Now you could also create a new plugin that implements the feature for logz.io if you want.